src/Explorer/Explorer.tsx

```diff
@@ -2,6 +2,7 @@ import { Link } from "@fluentui/react/lib/Link";
 import * as ko from "knockout";
 import React from "react";
 import _ from "underscore";
+import shallow from "zustand/shallow";
```

Comment:
why do we need this

Reply:
This is a third party library which we are using for subscribe/unsubscribe things, in my case I had a scenario where I need to  subscribing to same function with two different variables. So for variable comparison this the param that needed to be passed by library